### PR TITLE
add setproperties to createsheet

### PR DESCRIPTION
**Pull request by DnlLrssn**
**Checklist**

- [ ] tests added
- [x] commits conform to the [Commit message format](https://github.com/qlik-oss/gopherciser/blob/master/.github/CONTRIBUTING.md#commit)
- [ ] documentation updated

**Squash commit message**

Set two properties on sheet after creating it

**Info**

Optional informative text (not to be included in commit message) relevant to reviewers.

### CODE REVIEW

**Review by ashishkumarqlik — APPROVED** [review]
lgtm

File: scenario/createsheet.go [hunk 1]
```diff
@@ -66,6 +67,32 @@ func (settings CreateSheetSettings) Execute(sessionState *session.State,
 			return sessionState.IDMap.Add(settings.ID, genObj.GenericId, sessionState.LogEntry)
 		}
 
+		prop, err := genObj.GetPropertiesRaw(ctx)
+		if err != nil {
+			return err
+		}
+
+		// client updates two parameters, do this safely jsonparser as using a struct might "lose" values
```
Comment (SeBBBe) [inline scenario/createsheet.go:75]: Interesting 🤔  We might create a helper for this if it becomes more commonplace to do this kind of thing, for now it's great like it is!

**Review by SeBBBe — APPROVED** [review]

File: scenario/createsheet.go [hunk 2]
```diff
@@ -66,6 +67,32 @@ func (settings CreateSheetSettings) Execute(sessionState *session.State,
...
+		_, err = jsonparser.GetString(prop, "gridResolution")
+		if err == jsonparser.KeyPathNotFoundError {
+			prop, err = jsonparser.Set(prop, []byte(`"small"`), "gridResolution")
+			if err != nil {
+				return err
+			}
+		}
```
Comment (atluq) [inline scenario/createsheet.go:82]: Should we return errors that are not `jsonparser.KeyPathNotFoundError`?

Comment (DnlLrssn) [reply scenario/createsheet.go:82]: hmm yes, good point

Comment (DnlLrssn) [reply scenario/createsheet.go]: fixed

**Pull request merged**